v3/newrelic/app_run_test.go

```diff
@@ -382,27 +383,9 @@ func TestConfigurableTxnEvents_notInCollResponse(t *testing.T) {
 	expected := 10
 	cfg := config{Config: defaultConfig()}
 	cfg.TransactionEvents.MaxSamplesStored = expected
-	result := newAppRun(cfg, reply).MaxTxnEvents()
-	if result != expected {
-		t.Errorf("Unexpected max number of txn events, expected %d but got %d", expected, result)
-	}
-}
-
-func TestConfigurableTxnEvents_configMoreThanMax(t *testing.T) {
-	h, err := internal.UnmarshalConnectReply([]byte(
-		`{"return_value":{
-			"event_harvest_config": {
-				"report_period_ms": 10000
-			}
-        }}`), internal.PreconnectReply{})
-	if nil != err {
-		t.Fatal(err)
-	}
-	cfg := config{Config: defaultConfig()}
-	cfg.TransactionEvents.MaxSamplesStored = internal.MaxTxnEvents + 100
-	result := newAppRun(cfg, h).MaxTxnEvents()
-	if result != internal.MaxTxnEvents {
-		t.Errorf("Unexpected max number of txn events, expected %d but got %d", internal.MaxTxnEvents, result)
```

Comment:
Why was this test removed?

Reply:
Removed it at first because I was no longer using the MaxTxnEvents() function but forgot to add it back in, thank you for catching!